docs/SPIR-V.rst

```diff
@@ -2338,3 +2338,6 @@ either because of no Vulkan equivalents at the moment, or because of deprecation
 * Applying ``row_major`` or ``column_major`` attributes to a stand-alone matrix will be
   ignored by the compiler because ``RowMajor`` and ``ColMajor`` decorations in SPIR-V are
   only allowed to be applied to members of structures. A warning will be issued by the compiler.
+* The Hull shader ``partitioning`` attribute may not have the ``pow2`` value. The compiler
```

Comment:
+1

We should also update the code to give a more clear error message for `pow2`: `pow2 partitioning scheme is not supported since no equivalent in Vulkan`.

https://github.com/Microsoft/DirectXShaderCompiler/blob/cb4e570b82586088820f812403bb352c577c5306/tools/clang/lib/SPIRV/SPIRVEmitter.cpp#L6716-L6732